apps/myaccount/src/configs/app.ts

```diff
@@ -163,6 +164,8 @@ export class Config {
                 window["AppUtils"]?.getConfig()?.ui?.isMultipleEmailsAndMobileNumbersEnabled,
             isPasswordInputValidationEnabled: window["AppUtils"]?.getConfig()?.ui?.isPasswordInputValidationEnabled,
             isProfileUsernameReadonly: window["AppUtils"]?.getConfig()?.ui?.isProfileUsernameReadonly,
+            primaryUserStoreDomainName: window[ "AppUtils" ]?.getConfig()?.ui?.primaryUserStoreDomainName?.toUpperCase()
```

Comment:
Is it essential to convert the primary userstore name supplied by the config to upper case?

Reply:
Yes, otherwise some string comparisons may fail in the console. Since we are using uppercase domain names in the backend, it’s better to maintain consistency.

Reply:
Ack